src/main/java/uk/gov/register/core/Entry.java

```diff
@@ -53,7 +53,7 @@ public Integer getEntryNumber() {
 
     @JsonProperty("item-hash")
     public String getItemHash() {
-        return "sha-256:" + sha256hex;
+        return hashValue.toString();
```

Comment:
We can now remove this method entirely and put `@JsonProperty("item-hash")` and `@JsonSerialize(using = ToStringSerializer.class)` on the `getSha256hex()` method. Jackson should then call `toString()` behind the scenes.